### PR TITLE
Add octagon and parallelogram shapes

### PR DESCRIPTION
**Pull request by Nikpolik**
This pull request adds two new shapes

### CODE REVIEW

File: src/main/webapp/Octagon.jsp [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<head>
+<meta charset="ISO-8859-1">
+<title>Hexagon</title>
+</head>
+<body bgcolor="FEF9E7">
+<font face="verdana">
+	<h1>Hello!</h1>
+
+	<p>
+	<p>This is a <font color="blue">Blue Hexagon</font>.
```
Comment (ithanos460) [inline src/main/webapp/Octagon.jsp]: Replace Hexagon with Octagon

File: src/main/webapp/Parallelogram.jsp [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Square</title>
+</head>
+<body bgcolor="FEF9E7">
+<font face="verdana">
+	<h1>Hello!</h1>
+
```
Comment (ithanos460) [inline src/main/webapp/Parallelogram.jsp]: Please delete the empty line

**Review by ithanos460 — CHANGES_REQUESTED** [review]

File: src/main/webapp/Parallelogram.jsp (hunk 2 above)
Comment (Nikpolik) [reply src/main/webapp/Parallelogram.jsp]: Fixed in 31d38fb

File: src/main/webapp/Octagon.jsp (hunk 1 above)
Comment (Nikpolik) [reply src/main/webapp/Octagon.jsp]: Fixed in 5eedebd

**Review by ithanos460 — APPROVED** [review]

**Pull request merged**